lib/conditionals.md

```diff
@@ -5,8 +5,12 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+Conditionals are how a progrom decides what to do when it runs, based on a certain condition.
+For example, in an if statement, the computer will decide if a statement is true or false. If the statement is true, the code will run.
```

Comment:
I'm being a little nit-picky here but _what_ code will run if the conditional statement is true?